mne/channels/layout.py

```diff
@@ -1000,6 +1000,83 @@ def _merge_grad_data(data, method='rms'):
     return data.reshape(data.shape[:1] + orig_shape[1:])
 
 
+def _combine_meg_grads(inst, method='rms'):
+    """Combine pairs of gradiometer channels."""
+    # TODO:
+    # - modify units depending on method
+    # - need to handle noise_cov? projections?
+    # - currently assumes inst only contains 'grads'
+    # - outsource return instance creation to specific methods
+
+    picks = _pair_grad_sensors(inst.info, topomap_coords=False, exclude=[])
+    picks = np.array(picks).reshape(-1, 2)  # now [npairs x 2]
+    new_chs = list()
+    new_bads = list()
+    for idx in range(picks.shape[0]):
```

Comment:
you can do:

```suggestion
    for pair in picks:
```

then do below